.github/workflows/ci.yml

```diff
@@ -101,44 +101,46 @@ jobs:
           files: ./coverage.out
           verbose: true
 
+  go-integration-setup:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/alicenet-config
+      - name: npm Compile
+        working-directory: ./bridge
+        shell: bash
+        run: npm run compile
```

Comment:
`npm run compile` should have already run as part of `./.github/actions/alicenet-config`.

From a simplicity standpoint, I think we can pay the cost of `npm ci && npm run compile` (should only be 90-120 seconds), as the go integration tests are not going to be the long pole for CI at this point and there's a _lot_ of improvement we can do within the go code to speed things along.